scheduler/scheduler_system.go

```diff
@@ -190,17 +190,6 @@ func (s *SystemScheduler) process() (bool, error) {
 		return true, nil
 	}
 
-	// If the limit of placements was reached we need to create an evaluation
-	// to pickup from here after the stagger period.
-	if s.limitReached && s.nextEval == nil {
```

Comment:
AFAICT nextEval is now always `nil` ... can we just remove it? I removed it on main and everything compiled, but I hit a test failure.